src/hotspot/share/gc/shenandoah/shenandoahHeapRegion.cpp

```diff
@@ -360,25 +361,29 @@ void ShenandoahHeapRegion::make_committed_bypass() {
 }
 
 void ShenandoahHeapRegion::reset_alloc_metadata() {
```

Comment:
Do we need to make these atomic because we now increment asynchronously from within mutator CAS allocations?  Before, they were only adjusted while holding heap lock?  I'm wondering if add-with-fetch() or CAS() would be more/less efficient than AtomicAccess::stores.  Can we test the tradeoffs?